services/like.go

```
@@ -410,6 +412,144 @@ func (ls *LikeService) StartUpdateTicker() {
+	return FolloweeMessages, nil
+}
+
+// 获取用户关注数和粉丝数
+func (ls *LikeService) GetFollowCount(UserID string) (int, int, error) {
+	var count1, count2 int
+	key1 := "User:followers"
+	key2 := "User:followees"
+	followerscount, err1 := utils.Client.HGet(utils.Ctx, key1, UserID).Result()
+	followeescount, err2 := utils.Client.HGet(utils.Ctx, key2, UserID).Result()
+	if err1 == redis.Nil {
+		count1 = 0
```

golangci-lint (1.62.2)

ineffectual assignment to count1

(ineffassign)